CodeStarterCamp_Week2/lotto.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  lotto.swift
+//  CodeStarterCamp_Week2
+//
+//  Created by 양주원 on 2023/03/15.
+//
+
+import Foundation
+
+
+func winLottoNumbers() -> Set<Int> {
+    var winningNumberSet: Set<Int> = Set<Int>()
+        while winningNumberSet.count < 6 {
+            winningNumberSet.insert(Int.random(in: 1..<45))
+    }
+    return winningNumberSet
+}
+
+func checkWinMyNumbers(winLottoNumbers: Set<Int>) {
+    let myLottoNumbers = Set(myLottoNumbers)
+    let sameNumbers: Set<Int> = myLottoNumbers.intersection(winLottoNumbers)
+    let arraySameNumbers = Array(sameNumbers)
+    if arraySameNumbers.isEmpty {
+        print("아쉽지만 겹치는 번호가 없습니다.")
+    }
+    else {
+        print("축하합니다! 겹치는 번호는", terminator: "")
+        for row in arraySameNumbers {
```

Comment:
for문 내의 변수명의 경우, 배열의 각 항목을 나타내는 네이밍을 사용하시는 것이 좋습니다.
```suggestion
        for number in arraySameNumbers {
```